lib/tether/toolbox.go

```diff
@@ -219,25 +219,30 @@ func (t *Toolbox) halt() error {
 
 // externalIP attempts to find an external IP to be reported as the guest IP
 func (t *Toolbox) externalIP() string {
-	netif, err := net.InterfaceByName("client")
+	l, err := netlink.LinkByName("client")
```

Comment:
This should *not* be put in lib/portlayer if it's going to be referenced in lib/tether. That will again cause transitive inclusion of a vast amount of dead code.

We should consider moving this `vic-init` specific piece of logic into cmd/vic-init